### ISSUE TITLE
I have a question

### ISSUE THREAD

**Issue by ERENPROO** [AOF-Dev/MCinaBox#1129]
When I launch 1.18.2, it just gives a white screen. Is there any solution?

**Comment by ERENPROO** [comment]
Ummmm...

**Comment by 2019Z2** [comment]
I don't think so, but you can use PojavLauncher , a new launcher based on this app

**Comment by artdeell** [comment]
PojavLauncher is not based on MCinaBox

сб, 12 мар. 2022 г., 18:57 2019Z2 ***@***.***>:

**Comment by 2019Z2** [comment]
But it's based on the Boardwalk app

**Comment by ERENPROO** [comment]
> I don't think so, but you can use PojavLauncher , a new launcher based on this app

PojavLauncher crashes always, this is because of im using MCinaBox
